src/pages/NftDetailsScreen.vue

```diff
@@ -43,7 +44,7 @@ const nftAttributes: ComputedRef<NftAttributes[]> = computed(() => {
 })
 
 function openNftModal() {
-  router.push({ name: 'SendNfts', params: { ...props } })
+  router.push({ name: 'SendNfts', query: props })
```

Comment:
Why was this change required?

Reply:
because params not working for some reason